src/js/common/orderedStringify.js

```diff
@@ -0,0 +1,59 @@
+/* globals require */
+var fluid  = fluid  || {};
+(function (fluid) {
+    // TODO: pull this up into Infusion if we need to use this type of object hashing more broadly.
+    "use strict";
+
+    if (!fluid.identity) {
```

Comment:
Funny way of checking for Infusion - probably cleanest to move this up to the outer level where we attempt to allocate the fluid global in the first place. I believe that is the form of pattern we use elsewhere.

Reply:
I standardised to the usual check for fluid itself and kept the `fluid.require` pattern for the non-fluid dependencies.